prog/main.go

```diff
@@ -138,6 +142,41 @@ type appFlags struct {
 	consulInf       string
 }
 
+type containerLabelFiltersFlag struct {
+	apiTopologyOptions []app.APITopologyOption
+	exclude            bool
+}
+
+func (c *containerLabelFiltersFlag) String() string {
+	return fmt.Sprint(c.apiTopologyOptions)
+}
+
+func (c *containerLabelFiltersFlag) Set(flagValue string) error {
+	newAPITopologyOption, err := c.toAPITopologyOption(flagValue)
```

Comment:
You should return the error if not `nil` empty after this point. Not at the end.